.vscode/settings.json

```diff
@@ -0,0 +1,5 @@
+// Place your settings in this file to overwrite default and user settings.
```

Comment:
I don't like adding editor specific files to .gitignore (cause we'd need to do that for every editor which creates them) ;| Can't VS be configured to keep those files somewhere else?